lib/registries/addon/index/controller.ts

```diff
@@ -22,19 +20,14 @@ export default class Index extends Controller {
 
     @task({ withTestWaiter: true, on: 'init' })
     getRecentRegistrations = task(function *(this: Index) {
-        const [recentResults, totalResults]: Array<SearchResults<ShareRegistration>> = yield RSVP.all([
-            this.shareSearch.registrations(new SearchOptions({
-                order: new SearchOrder({ display: '', ascending: false, key: 'date' }),
-                query: config.indexPageRegistrationsQuery,
-                size: 5,
-            })),
-            this.shareSearch.registrations(new SearchOptions({
-                size: 0,
-            })),
-            this.store.findAll('registration-schema'),
-        ]);
-        this.set('recentRegistrations', recentResults.results);
-        this.set('searchableRegistrations', totalResults.total);
+        const recentRegistrations = yield this.store.query('registration', {
+            filter: {
+                id: config.indexPageRegistrationsQuery,
+            },
+            sort: '-date_modified',
+            embed: 'contributors',
```

Comment:
could use the `bibliographic_contributors` relation instead and avoid the `filterBy('bibliographic')`